### PR TITLE
LDP containers are the correct type

### PR DESCRIPTION
**Pull request by ndushay**
(all are Basic Container except for /t and /b, per #132)

connected to #132

### CODE REVIEW

**Comment by ndushay** [comment]
@darrenleeweber

File: app/services/triannon/ldp_writer.rb [hunk 1]
```diff
@@ -167,6 +167,7 @@ def create_base
       g = OA::Graph.new(g)
       g.remove_non_base_statements
       g.make_null_relative_uri_out_of_blank_node
+      g << [RDF::URI.new, RDF.type, RDF::Vocab::LDP.BasicContainer]
```
Comment (darrenleeweber) [inline app/services/triannon/ldp_writer.rb]: Does OA::Graph support this syntax sugar?

**Comment by darrenleeweber** [comment]
Specs failing, so review specs before merging.  (Don't know enough about FC4 LDP to comment on the substance of this PR.)

File: app/services/triannon/ldp_writer.rb (hunk 1 above)
Comment (ndushay) [inline app/services/triannon/ldp_writer.rb]: OA::Graph is just a wrapper around RDF::Graph, so you can do any RDF::Graph method on an OA::Graph object.

**Comment by ndushay** [comment]
yes, will get specs passing.  Looks like vcr cassette woes.

**Pull request merged**